.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "modules/accord"]
 	path = modules/accord
-	url = https://github.com/apache/cassandra-accord.git
-	branch = trunk
+	url = https://github.com/belliottsmith/cassandra-accord.git
```

Comment:
I'm sure you had it in mind, but just for completeness, reminding to switch branch.